dotcom-rendering/cypress/integration/parallel-1/article.elements.spec.js

```diff
@@ -109,7 +109,7 @@ describe('Elements', function () {
 			cy.scrollTo(0, 4500);
 
 			// Wait for hydration
-			cy.get('gu-island[name=EmbedBlockComponent]')
+			cy.get('gu-island[name=EmbedBlockComponent]', { timeout: 30000 })
```

Comment:
Guessing this is unrelated but curious why it was added?!

Reply:
I can’t figure it out… 🤷 

I’ll try rebasing.